### PR TITLE
{chem}[foss/2020b] PLUMED v2.7.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/71b33d774ae491cbedc474ae8bee682f for a full test report.

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.7.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('GSL', '2.6'),
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Boost', '1.74.0'),
+]
+
+preconfigopts = 'env FC=$MPIF90 LIBS="$LIBLAPACK $LIBS" '
+configopts = '--exec-prefix=%(installdir)s --enable-gsl --enable-modules=all --enable-python i'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.7.0-foss-2020b.eb:30]: I guess the `i` is a typo, but how come it gets ignored?

Comment (Micket) [reply easybuild/easyconfigs/p/PLUMED/PLUMED-2.7.0-foss-2020b.eb:30]: the good old vim "i" for insertmode typo. Configure scripts aren't typically all that robust? It getting ignored doesn't surprise me much.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @Micket!

**Pull request merged**